dev-java/jetty-util/jetty-util-9.3.5.20151012.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+JAVA_PKG_USE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Jetty's Utils"
+
+MY_PN="jetty"
+MY_PV="${PV/2015/v2015}"
+MY_P="${MY_PN}-${MY_PV}"
+
+SLOT="$(get_version_component_range 1-2)"
```

Comment:
Really? Have you looked at eclasses? Do you not KNOW that it is brought in by the base java-utils-2.eclass. Let me show you since you clearly missed it...

/usr/portage/eclass/java-utils-2.eclass

inherit eutils versionator multilib

But you just had to comment because its my PR right...